examples/02-javascript-basics/bob.js

```diff
@@ -7,14 +7,58 @@ He says 'Fine. Be that way!' if you address him without actually saying anything
 He answers 'Whatever.' to anything else.
 */
 
-function hey () {
+function hey (convo) {
+  var questioned_at = 'Sure.';
+  var yelled_at = 'Whoa, chill out!';
+  var yelled_question_at = 'Calm down, I know what I\'m doing!';
+  var not_say_any = 'Fine';
+  var whatever = 'Whatever.';
```

Comment:
@jlabausa good use of variables. You can also use an object here so they are in one data structure. Like how most translations are implemented:

```js
{
  'questioned_at': 'Sure.'
}
```